drop_locations:
  "1"   :   " 4.3  ,   24.6    ,   0.0   "
  "2"   :   " 4.3  ,   23.0    ,   0.0   "
  "3"   :   " 4.3  ,   21.4    ,   0.0   "
  "4"   :   " 4.3  ,   19.8    ,   0.0   "
  "5"   :   " 4.3  ,   18.2    ,   0.0   "
  "6"   :   " 4.3  ,   16.6    ,   0.0   "
  "7"   :   " 4.3  ,   15.0    ,   0.0   "
  "8"   :   " 4.3  ,   13.4    ,   0.0   "
  "9"   :   " 4.3  ,   11.8    ,   0.0   "
  "10"  :   " 4.3  ,   10.2    ,   0.0   "
  "11"  :   " 4.3  ,   8.6     ,   0.0   "
  "22"  :   " 9.7  ,   24.6    ,   3.14   "
  "21"  :   " 9.7  ,   23.0    ,   3.14   "
  "20"  :   " 9.7  ,   21.4    ,   3.14   "
  "19"  :   " 9.7  ,   19.8    ,   3.14   "
  "18"  :   " 9.7  ,   18.2    ,   3.14   "
  "17"  :   " 9.7  ,   16.6    ,   3.14   "
  "16"  :   " 9.7  ,   15.0    ,   3.14   "
  "15"  :   " 9.7  ,   13.4    ,   3.14   "
  "14"  :   " 9.7  ,   11.8    ,   3.14   "
  "13"  :   " 9.7  ,   10.2    ,   3.14   "
  "12"  :   " 9.7  ,   8.6     ,   3.14   "
  
pick_locations:
  "P3"  :   "10.0   ,   3.5     ,   1.57  "
  "P2"  :   "7.0    ,   3.5     ,   1.57  "
  "P1"  :   "4.0    ,   3.5     ,   1.57  "

robot_locations:
  "R1"  :   "12.5   ,   3.0     ,   3.14  "
  "R2"  :   "12.5   ,   5.0     ,   3.14  "
  "R3"  :   "12.5   ,   7.0     ,   3.14  "
  "R4"  :   "1.5    ,   3.0     ,   0.00  "
  "R5"  :   "1.5    ,   5.0     ,   0.00  "
  "R6"  :   "1.5    ,   7.0     ,   0.00  "

special_behaviour:
  "W"   :   "2"
  "I"   :   "5"
  "L"   :   "R"
  "T"   :   "T"
  "D"   :   "T"